salt/grains.j2

```diff
@@ -75,18 +75,14 @@ sbd_disk_device: "{{ sbd.device if common.shared_storage_type == "shared-disk" e
 {% endif %}
 
 
-iscsi_enabled: {{ jsonify(true) if common.shared_storage_type == "iscsi" else jsonify(false) }}
-
-iscsi_name: "{{  iscsi.name if "iscsi" in env }}"
-
-iscsi_ip: "{{  iscsi.private_ip if "iscsi" in env }}"
-
 qdevice_enabled: {{ jsonify(qdevice.enabled) if "qdevice" in env else jsonify(false) }}

```

Comment:
What happens now with this `qdevice_enabled` and the others?
Are they still used?